ruby/vampires.rb

```diff
@@ -62,6 +62,7 @@
 		possibility_3 = true
```

Comment:
on line 15 consider this 

with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.now.year`